tensor/datafs/list.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package datafs
+
+import "strings"
+
+const (
+	Short = false
+	Long  = true
+
+	DirOnly   = false
+	Recursive = true
```

Comment:
These bool constants are undocumented and ambiguously named. Moreover, the general bool constant paradigm is unideal, and given that you need to make this a cli config struct at some point anyway, I would strongly reconsider this approach.